create-astronomical-data

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+from datetime import date
+from gzip import open
+from json import dumps
```

Comment:
```suggestion
import datetime
import gzip
import json
```
Importing whole modules rather than the objects they contain usually leads to more readable and testable code.
(note that is you import a submodule, you'd still do `from X.Y import Z` where Z is the submodule)